File: Stripe_Android/app/src/main/java/com/example/stripe_android/MainActivity.java
package com.example.stripe_android;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import com.android.volley.Request;
import com.android.volley.RequestQueue;
import com.android.volley.Response;
import com.android.volley.VolleyError;
import com.android.volley.toolbox.StringRequest;
import com.android.volley.toolbox.Volley;
import com.stripe.android.paymentsheet.PaymentSheet;

import java.util.HashMap;
import java.util.Map;

public class MainActivity extends AppCompatActivity {

    PaymentSheet paymentSheet;
    String paymentIntentClientSecret;
    PaymentSheet.CustomerConfiguration configuration;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Button button = findViewById(R.id.pay_now);
        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

            }
        });
        paymentSheet = new PaymentSheet(this, onPaymentSheetResult());
    }
    private void onPaymentSheetResult()

    public void fetchApi(){
        RequestQueue queue = Volley.newRequestQueue(this);
        String url ="https://api.jsonbin.io/v3/b/6435c73eace6f33a22093f79";

        StringRequest stringRequest = new StringRequest(Request.Method.POST, url,
                new Response.Listener<String>() {
                    @Override
                    public void onResponse(String response) {

                    }
                }, new Response.ErrorListener() {
            @Override
            public void onErrorResponse(VolleyError error) {

            }
        }){
            protected Map<String, String> getParams(){
                Map<String, String> paramV = new HashMap<>();
                paramV.put("param", "abc");
                return paramV;
            }
        };
        queue.add(stringRequest);
    }
}

File: FirstKotlin/app/src/main/java/com/example/firstkotlin/MainActivity.kt
package com.example.firstkotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.EditText
import android.widget.TextView

        private lateinit var editTextName: EditText
        private lateinit var textViewName: TextView
        private lateinit var buttonSave: Button


class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        editTextName = findViewById(R.id.editTextName)
        textViewName = findViewById(R.id.textViewName)
        buttonSave =findViewById(R.id.buttonSave)
        buttonSave.setOnClickListener {
            val name = editTextName.text.toString().trim()
            textViewName.text = "Hello, $name"
        }
    }
}